superset/views/core.py

```diff
@@ -105,8 +105,8 @@
     json_success,
     SupersetModelView,
 )
+from .chart import views as chart_views
 from .dashboard import views as dash_views
-from .dashboard.filters import DashboardFilter
 from .database import views as in_views
 from .utils import (
```

Comment:
Same here regarding absolute vs relative imports.

Reply:
Will make a separate PR to address all relative imports on views